AnkiDroid/src/main/java/com/ichi2/anki/utils/ext/Collection.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2025 Brayan Oliveira <brayandso.dev@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.ichi2.anki.utils.ext
+
+import androidx.annotation.CheckResult
+import anki.collection.OpChangesWithCount
+import com.ichi2.anki.Flag
+import com.ichi2.libanki.Collection
+
+/** Change the flag color of the specified cards. flag=0 removes flag. */
```

Comment:
No longer needed now we use `Flag`

```suggestion
/** Change the flag color of the specified cards */
```